Settings-panel (#44)
iXtenda/iVP-gitbook: home/ivp-planning/settings/text-objects-settings.md

# Text Objects

> The text object settings offer several options to adjust the behaviour of the Text Object asset. The text object settings can be opened via the [settings panel](../user-interface/settings-panel.md).

Reviewer: ```suggestion
The text object settings offer several options to adjust the behaviour of the Text Object from the library.
```

![](../../../.gitbook/assets/iVP_settings_text_objects.jpg)

## The options explained:

* Show Objects: Toggles object visibility
* Show Text Contents: Toggles text visibility

> * Set Objects Orientation to Horizontal: Converts text objects to 2D display in 3D-View

Reviewer: I find the wording confusing

> * Object looks at Camera: Text tracks with 3D-view camera

Reviewer: ```suggestion
* Object looks at Camera: The text will always face in the direction of the camera in the 3D panel
```
